bundles/org.openhab.binding.hapero/src/main/java/org/openhab/binding/hapero/internal/handler/HaperoBridgeHandler.java

```diff
@@ -0,0 +1,615 @@
+
+                dp = DirectoryPoller.newBuilder().addPolledDirectory(polledDirectory)
+                        .addListener(new HaperoFtpEventListener()).enableFileAddedEventsForInitialContent()
+                        .setPollingInterval(config.pollingInterval, TimeUnit.SECONDS).start();
+
+                /* Start Timeout monitoring for the FTP Watcher */
+                startFtpTimeout();
+            } else if (config.accessMode.contentEquals(HaperoBindingConstants.CONFIG_ACCESS_FILESYSTEM)) {
+                /* Start a file watcher */
+                try {
+                    watchPath = Path.of(config.fileStoragePath);
+                } catch (InvalidPathException e) {
+                    logger.debug("Exception in Init: {}", e.getMessage());
+                    faultOcurred = true;
+                    watchPath = null;
+                    errorMsg = "@text/offline.monitoring.failed";
+                }
+
+                if (watchPath != null) {
+                    try {
+                        watchService = FileSystems.getDefault().newWatchService();
+                        watchPath.register(watchService, StandardWatchEventKinds.ENTRY_MODIFY);
+                    } catch (Exception e) {
+                        logger.debug("Exception in Init: {}", e.getMessage());
+                        faultOcurred = true;
+                        errorMsg = "@text/offline.monitoring.failed";
+                    }
+                } else {
+                    logger.debug("Failed to create watchservice");
+                    faultOcurred = true;
+                    errorMsg = "@text/offline.monitoring.failed";
+                }
+
+                /* Start the File Watcher Runnable if a Watcher could be created */
+                if (!faultOcurred) {
+                    fileWatcherJob = scheduler.schedule(fileWatcherTask, 0, TimeUnit.SECONDS);
+                }
+            }
+
+            if (!faultOcurred) {
+                /*
+                 * If a watcher could be created, start a discovery.
+                 * The Thing status will be set to ONLINE after the first successful update
+                 */
+                updateStatus(ThingStatus.UNKNOWN);
+
+                scheduler.execute(() -> {
+                    updateThings();
+
+                    if (discoveryService != null) {
+                        discoveryService.startScan();
+                    }
+                });
+                logger.trace("Bridge initialized");
+            } else {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, errorMsg);
+            }
+        } else {
+            /* A Watcher could not be created, set Thing to OFFLINE. Manual restart is required. */
+            cancelAllJobs();
+
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, errorMsg);
```

Comment:
`errorMsg` has not been set here. Maybe https://github.com/openhab/openhab-addons/pull/14186#discussion_r1153531917 was misinterpreted? I meant only the logging was not needed, but it appears the error message was also removed.

Reply:
OK, fixed